crates/cli/src/commands/add.rs

```diff
@@ -23,6 +23,9 @@ pub struct AddCommandArgs {
     /// Install the specified packages as optionalDependencies.
     #[arg(short = 'O', long = "save-optional", group = "dependency_group")]
     save_optional: bool,
+    /// Install the specified packages as optionalDependencies.
```

Comment:
This comment is wrong. 

Using --save-peer will add one or more packages to peerDependencies and install them as dev dependencies.